### PR TITLE
chore: Implement pr-prevent-kustomization GHA

### PR DESCRIPTION
**Pull request by TeodorSAP**
## Description

Changes proposed in this pull request (what was done and why):

- Add a new GHA that prevents modifying `kustomization.yaml`

Changes refer to particular issues, PRs or documents:

- 

## Traceability
- [ ] The PR is linked to a GitHub issue.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.
- [ ] If the change is user-facing, the documentation has been adjusted.
- [ ] The feature is unit-tested.
- [ ] The feature is e2e-tested.

### CODE REVIEW

File: .github/workflows/pr-github-checks.yml [hunk 1]
```diff
@@ -70,6 +70,16 @@ jobs:
...
+    steps:
+      - name: Prevent kustomization.yaml changes
+        uses: xalvarez/prevent-file-change-action@v1.6.0
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          pattern: .\config\kustomization.yaml
+          allowNewFiles: false
+
   PR-Github-Checks-Success:
     needs: [pr-milestone-check, pr-label-check, pr-title-check]
```
Comment (k15r) [inline .github/workflows/pr-github-checks.yml:84]: please add also to the "needs" here

**Comment by k15r** [comment]
Really cool idea! 👍

**Comment by TeodorSAP** [comment]
> Really cool idea! 👍

It was Stas's idea =))

**Review by k15r — APPROVED** [review]

**Review by k15r — CHANGES_REQUESTED** [review]
please remove the pull_request trigger

**Review by k15r — APPROVED** [review]

**Pull request merged**